presto/presto.sh

```diff
@@ -22,17 +22,12 @@ export PATH=/usr/bin:$PATH
 readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
 readonly PRESTO_MASTER_FQDN="$(/usr/share/google/get_metadata_value attributes/dataproc-master)"
 readonly WORKER_COUNT=$(/usr/share/google/get_metadata_value attributes/dataproc-worker-count)
-if [[ -d /usr/local/share/google/dataproc/lib ]]; then
-  readonly CONNECTOR_JAR="$(find /usr/local/share/google/dataproc/lib -name 'gcs-connector-*.jar')"
-else
-  readonly CONNECTOR_JAR="$(find /usr/lib/hadoop/lib -name 'gcs-connector-*.jar')"
-fi
-readonly PRESTO_VERSION='0.206'
+readonly PRESTO_MAJOR_VERSION='302'
```

Comment:
I could clean this up such that it's a substring a single variable that specifies the version.

Reply:
This is used to construct links for wget, so you can leave as is.

Reply:
Double quotes look better & more consistent

```suggestion
readonly PRESTO_MAJOR_VERSION="302"
```